### DISCUSSION TITLE
Bezier curves + Physics

### DISCUSSION THREAD

**Discussion by dmitrykolesnikovich** [dyn4j/dyn4j#228] [Q&A]
I do not find in samples any mention of applying physics properties to bezier curves. Is it possible with the current state of library?

**Comment by wnbittle** [comment]
Bézier curves are not provided in the library.  

If you wanted to use it as a collision shape, your best option would be to convert it into discrete line segments and use the Link shape.  

If you wanted to use the Bézier curve as part of a joint or constraint, its possible, but you'd need to build that yourself - there isn't a point on curve constraint.  The challenge with point on (arbitrary or bezier) curves is that it boils down to finding the closest point on a curve to a point.  That isn't hard, just not very efficient.
